client/src/helpers/nova/hooks/useNftAddressState.ts

```diff
@@ -56,8 +62,15 @@ export const useNftAddressState = (address: NftAddress): [INftAddressState, Reac
         initialState,
     );
 
-    const { nftOutput, isLoading: isNftDetailsLoading } = useNftDetails(network, address.nftId);
-    const { totalBalance, availableBalance } = useAddressBalance(network, state.addressDetails, nftOutput);
+    const { nftOutput, nftOutputMetadata, isLoading: isNftDetailsLoading } = useNftDetails(network, address.nftId);
+    const { manaRewards } = useOutputManaRewards(network, nftOutputMetadata?.outputId ?? "");
```

Comment:
Same as AnchorOutput,  we can remove `manaRewards`  for nft output because the `getRewards` returns the totally available Mana rewards of an account or delegation output.